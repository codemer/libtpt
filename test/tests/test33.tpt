There are @size(myarray) objects in myarray.
@foreach obj (myarray) {
${obj}
}
