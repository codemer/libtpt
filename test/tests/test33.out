There are 4 objects in myarray.
value1
value2
value3
value4
